hyperion_http.gemspec

```diff
@@ -18,6 +18,7 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
+  spec.required_ruby_version = '~> 2.2'
```

Comment:
Because it's the minimum version of ruby required by hyperion (due to the `rack` dependency).

On second thought, this isn't buying us anything. I'll remove it and let bundler do the work of dynamically figuring out what the minimum supported version of ruby is based on dependencies.